### PR TITLE
[service.subtitles.titlovi-com-official] 1.0.0

### PR DESCRIPTION
**Pull request by tturkalj**
### Description
Titlovi.com offical subtitle addon for Kodi.
Addon is using new Titlovi.com API instead scraping website data.
In order to use addon users need to be registered on Titlovi.com and input their login data in addon settings.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.subtitles.titlovi-com-official/addon.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <language></language>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <website>http://www.titlovi.com</website>
+        <email>tturka33 [at] gmail {dot} com</email>
+        <assets>
+            <icon>icon.png</icon>
+            <fanart>fanart.jpg</fanart>
+			<screenshot>resources/media/screenshot1.png</screenshot>
+            <screenshot>resources/media/screenshot2.png</screenshot>
+            <screenshot>resources/media/screenshot3.png</screenshot>
```
Comment (Rechi) [inline service.subtitles.titlovi-com-official/addon.xml]: those filenames are case sensitive (your screenshot have a capital extension)

Comment (MartijnKaijser) [reply service.subtitles.titlovi-com-official/addon.xml]: converting them to a jpg format is also appreciated as they are smaller

**Comment by MartijnKaijser** [comment]
Also move language to new folder layout
http://kodi.wiki/view/language_support#Layout_of_the_language_folders

File: service.subtitles.titlovi-com-official/addon.xml (hunk 1 above)
Comment (tturkalj) [reply service.subtitles.titlovi-com-official/addon.xml]: Thanks for the review, i figured that extensions should be lowercase...

**Comment by tturkalj** [comment]
thanks on the link

**Comment by tturkalj** [comment]
Can you please confirm if this is correct path for strings file:
"service.subtitles.titlovi-com-official\resources\language\resource.language.en_us\strings.po"

Because its not working in Kodi v17.6, strings are not showing. Path is taken from the link above.

**Comment by tturkalj** [comment]
@MartijnKaijser @Rechi  just pinging in case you didnt get notification for my post above :)

**Comment by tturkalj** [comment]
@MartijnKaijser @Rechi  I have fixed all previous issues, can you please resume reviewing?

**Pull request closed without merging**